README.md

```diff
@@ -32,9 +26,11 @@ All scripts accept a help argument which displays all input flags. Use "python s
 
 You must be in the following NCI project groups to perform certain tasks with these scripts;
 
-* **ub8** - Data Access
+* **ub8** - Data Access (**ub8_admin** for write permissions)
 * **xc0** - To schedule Raijin jobs & edit scripts
 
+Alternatively, you could edit the various *.qsub files to run under a different project's Raijin allocation.
```

Comment:
Need to escape the `*` to avoid entering italics - either surround the word in backticks for a code literal or use a backslash, like:

    `*.qsub` or \*.qsub